core/src/main/scala/io/projectglow/vcf/TabixIndexHelper.scala

```diff
@@ -482,17 +482,24 @@ object TabixIndexHelper extends GlowLogging {
           if (offsetList.isEmpty) {
             None
           } else {
-            val (minOverBlocks, maxOverBlocks) = offsetList
-              .foldLeft((offsetList(0).getStartPosition, offsetList(0).getEndPosition)) {
-                case ((myStart, myEnd), e) =>
-                  (Math.min(myStart, e.getStartPosition), Math.max(myEnd, e.getEndPosition))
-              }
-            val blockRangeStart = Math.max(file.start, minOverBlocks >> 16) // Shift 16 bits to get
-            // file offset of the bin from bgzipped virtual file offset.
-            val blockRangeEnd = Math.min(file.start + file.length, (maxOverBlocks >> 16) + 0xFFFF)
-            // 0xFFFF is the maximum possible length of an uncompressed bin.
-            if (blockRangeStart <= blockRangeEnd) {
-              Some((blockRangeStart, blockRangeEnd))
+            // Do not return any files without a BGZIP header
```

Comment:
nit: Change the comment to: `Flag used to avoid returning any files without a BGZIP header`